api/test/handler.test.js

```diff
@@ -1,44 +1,31 @@
-import createDebug from 'debug'
-import { once } from 'node:events'
-import http from 'node:http'
 import { after, before, describe, it } from 'node:test'
 
 import { createPgPool, migrateWithPgClient } from '../../db/index.js'
-import { createHandler } from '../lib/handler.js'
-import { assertResponseStatus, getPort } from './test-helpers.js'
-
-const debug = createDebug('test')
+import { createApp } from '../lib/app.js'
+import { assertResponseStatus } from './test-helpers.js'
 
 describe('HTTP request handler', () => {
   /** @type {import('@filecoin-station/deal-observer-db').PgPool} */
   let pgPool
-  /** @type {http.Server} */
-  let server
+  /** @type {import('fastify').FastifyInstance} */
+  let app
   /** @type {string} */
   let baseUrl
 
   before(async () => {
     pgPool = await createPgPool()
     await migrateWithPgClient(pgPool)
 
-    const handler = createHandler({
+    app = createApp({
       pgPool,
-      logger: {
-        info: debug,
-        error: console.error,
-        request: debug
-      }
+      logger: false
```

Comment:
This is a "breaking" change. Before, tests printed info messages via the `debug` channel, thus keeping the test output clean. After your change, all info messages will be printed to the console, potentially polluting the test output.

Can we preserve the old behaviour please?

Reply:
I was concerned about this as well, but didn't look into it further out of laziness. I will fix it :)

Reply:
https://github.com/filecoin-station/deal-observer/pull/17/commits/36ef539a3123011c583f1574cceb323fbed95dd0